platforms/core-configuration/declarative-dsl-provider/src/integTest/groovy/org/gradle/internal/declarativedsl/settings/SoftwareTypeConventionIntegrationTest.groovy

```diff
@@ -285,22 +285,38 @@ class SoftwareTypeConventionIntegrationTest extends AbstractIntegrationSpec impl
         outputContains("""id = test\nbar = convention""")
     }
 
-    @NotYetImplemented
-    def "can configure build-level conventions in a declarative settings file and apply in a non-declarative project file"() {
+    def "can configure build-level conventions in a declarative settings file and apply in a non-declarative project file (#type build script)"() {
         given:
         withSoftwareTypePlugins().prepareToExecute()
 
-        file("settings.gradle.dcl") << getDeclarativeSettingsScriptThatSetsConventions(setAll("convention", "convention"))
+        file("foo").createDir()
+        file("bar").createDir()
```

Comment:
I don't think these are necessary because we automatically create the directories for TestFiles

Reply:
True.  Removed.